metaseq/file_io/common/__init__.py

```diff
@@ -741,6 +741,14 @@ def path_requires_pathmanager(self, path: str) -> bool:
                 return True
         return False
 
+    def supports_rename(self, path: str) -> bool:
+        # PathManager doesn't yet support renames
+        return not self.path_requires_pathmanager(path)
+
+    @staticmethod
+    def rename(src: str, dst: str):
```

Comment:
I believe this should be an instance method. `metaseq.file_io.PathManager` was previously a static wrapper around iopath's global instance, but I changed it to be a global instance as well as part of https://github.com/facebookresearch/metaseq/pull/441.

You should also be able to confirm these are working as expected by adding new/updated tests [here](https://github.com/facebookresearch/metaseq/blob/main/tests/file_io/test_native.py) and running `python -m tests.file_io.test_native`

Reply:
+1; might be even safer to call `supports_rename` from within `rename` too.

Reply:
Gotcha @davides, thanks. Also added the `supports_rename` guard in `rename` (raising a `ValueError` if it doesn't support rename - didn't want it silently fail. The only place `rename` is called in the codebase is immediately following a `supports_rename` check anyway, so this won't break anything elsewhere).